checks.d/rabbitmq.py

```diff
@@ -216,7 +221,10 @@ def get_stats(self, instance, base_url, object_type, max_detailed, filters, auth
                     continue
 
                 for p in regex_filters:
-                    if re.search(p, absolute_name):
+                    match = re.search(p, absolute_name)
+                    if match:
+                        if instance.get("tag_families", False) and match.groups():
```

Comment:
@rhwlo I missed that the `tag_families` option was also used here, sorry about that.

Could you also use `_is_affirmative` here?